specs/Media/doc/introduction.md

```diff
@@ -0,0 +1,10 @@
+# Media Data Models
+
+These data models describe the main entities involved in multimedia sensor basing on computer vision. This pattern is based on using computer vision or speech analysis techniques to transform a device camera into an advanced sensor. Media servers provide, off-the-shelf, some basic computer vision capabilities for detecting faces, detecting crowds, detecting car plates, detecting movement, etc. Note that most of these analyses lead to generation of data that takes the form of events (e.g. detected a specific face, detected a car plate, detected an intruder, etc.)
+
+You may want to manage these events as Context Information. These models represent those events.
+
+The main entities identified are:
+
++ [MediaEvent](../MediaEvent/doc/spec.md). It represents a base for all events raised by elements in the media server
++ [PlateDetectedEvent](../PlateDetectedEvent/doc/spec.md). It represents an Event when a plate is found in the video stream.
```

Comment:
where is the specification of the Entity Type PlateDetectedEvent? Is that a new entity type or should it just be one of the type names allowed by the `eventType`field?

Reply:
Missed doc update. Working on it...
I had simplified the MediaEvents to be generic enough so there isn't necessary to define different models for each  MediaEvent sub-type.